doc/nrf/links.txt

```diff
@@ -593,6 +606,13 @@
 .. _`Nordic Semiconductor's IoT cloud platform`: https://www.nordicsemi.com/Products/Cloud-services
 .. _`Securely Generating Credentials on the nRF9160`: https://docs.nrfcloud.com/Guides/GettingStarted/Devices#securely-generating-credentials-on-the-nrf9160
 
+.. ### Source: zigbeealliance.org & csa-iot.org
+
+.. _`Zigbee Alliance`:
+.. _`Connectivity Standards Alliance`: https://csa-iot.org
+.. _`Zigbee Specification`: https://zigbeealliance.org/developer_resources/
```

Comment:
I don't think you can find Zigbee specification under this address?

Reply:
There's the Zigbee BDB Specification v1.0 there. Still, it's not a new link, I've just grouped them together, so a potential change could trigger other edits outside of this PR. If you think it is wrong, I'll create a task for going through this and other Zigbee links.